python-checks/src/main/java/org/sonar/python/checks/AllBranchesAreIdenticalCheck.java

```diff
@@ -48,6 +49,7 @@ public class AllBranchesAreIdenticalCheck extends PythonSubscriptionCheck {
   private static final String CONDITIONAL_MESSAGE = "This conditional expression returns the same value whether the condition is \"true\" or \"false\".";
 
   private static final List<ConditionalExpression> ignoreList = new ArrayList<>();
+  public static final String SECONDARY_MESSAGE = "Redundant statements.";
```

Comment:
Thinking about it, I think `Duplicated statements.` would probably make more sense, given the statements themselves are not redundant (the conditional branching is). Wdyt?

Reply:
done